explorer/client/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -283,14 +283,30 @@ function TransactionView({ txdata }: { txdata: DataType }) {
         ],
     };
 
-    const validatorSignatureData = {
-        title: 'Validator Signatures',
-        content: txdata.authSignInfo.signature.map((validatorSign, index) => ({
-            label: `Signature #${index + 1}`,
-            value: validatorSign,
-            monotypeClass: true,
-        })),
-    };
+    let validatorSignatureData;
+    if (Array.isArray(txdata.authSignInfo.signature)) {
+        validatorSignatureData = {
+            title: 'Validator Signatures',
+            content: txdata.authSignInfo.signature.map(
```

Comment:
Just curious, why do we need this? Don't we wipe the whole DB anyways?

Reply:
yes, just doing this for precaution, we can eventually deprecate the legacy display